### PR TITLE
feat: setup Files tab in qa-mickey

### PR DESCRIPTION
**Pull request by m0nhawk**

### CODE REVIEW

File: qa-mickey.planx-pla.net/portal/gitops.json [hunk 1]
```diff
@@ -152,108 +146,165 @@
...
+          "leftIcon": "datafile",
+          "rightIcon": "download",
+          "fileName": "manifest.json"
+        }
+      ],
+      "table": {
+        "enabled": true,
+        "fields": [
+          "submitter_id",
+          "auth_resource_path",
```
Comment (mfshao) [inline qa-mickey.planx-pla.net/portal/gitops.json:286]: probably should not show this in table?

File: qa-mickey.planx-pla.net/etlMapping.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+mappings:
+  - name: qa-mickey_file
```
Comment (mfshao) [inline qa-mickey.planx-pla.net/etlMapping.yaml:2]: this is different with the guppy config in manifest.json?

File: qa-mickey.planx-pla.net/portal/gitops.json [hunk 3]
```diff
@@ -152,108 +146,165 @@
...
-            "Oncology_Primary.Lymphvascularinvasionl",
-            "Oncology_Primary.SeerSummaryStage2000",
-            "Oncology_Primary.PhysiciansStage",
-            "Oncology_Primary.DaysFromAnchorDateToMostdefinitivesurgDate",
-            "Oncology_Primary.ChemotherapyFac",
-            "Oncology_Primary.TumorSize",
-            "Oncology_Primary.SequenceNumber"
-          ]
+          "enabled": false,
+          "type": "manifest",
```
Comment (mfshao) [inline qa-mickey.planx-pla.net/portal/gitops.json:275]:
```suggestion
          "type": "file-manifest",
```

File: qa-mickey.planx-pla.net/portal/gitops.json [hunk 4]
```diff
@@ -152,108 +146,165 @@
...
-        "referenceIdFieldInResourceIndex": "case_id",
-        "referenceIdFieldInDataIndex": "Chicago_ID"
       }
     }
-  },
+  ],
   "showArboristAuthzOnProfile": true,
   "showFenceAuthzOnProfile": false
-}
+}
```
Comment (mfshao) [inline qa-mickey.planx-pla.net/portal/gitops.json:310]: new line?

File: qa-mickey.planx-pla.net/portal/gitops.json (hunk 1 above)
Comment (m0nhawk) [reply qa-mickey.planx-pla.net/portal/gitops.json]: True, removed it.

File: qa-mickey.planx-pla.net/portal/gitops.json [hunk 5]
```diff
@@ -152,108 +146,164 @@
...
+          "leftIcon": "datafile",
+          "rightIcon": "download",
+          "fileName": "manifest.json"
+        }
+      ],
+      "table": {
+        "enabled": true,
+        "fields": [
+          "project_id",
+          "auth_resource_path",
```
Comment (mfshao) [inline qa-mickey.planx-pla.net/portal/gitops.json:232]: I missed this one the last time 🤦

**Pull request merged**